### DISCUSSION TITLE
Deployment in Azure AKS

### DISCUSSION THREAD

**Discussion by royashiki** [open-metadata/OpenMetadata#8148] [Q&A]
I didn't find any specific resource about the deployment of OpenMetaData in Azure AKS Service, I would like to know how to properly do it, ie in AWS we have to create specific resources, do we have to the same for Azure, which are these resources?

**Comment by pmbrull** [comment]
Hi @royashiki, we are working on getting these docs ready. Pinging @akash-jain-10 for any updates

**Reply by fredericocotrim** [reply]
Any update about docs of deployment OpenMetaData in Azure AKS Service? @akash-jain-10

**Comment by pauloventurab** [comment]
I would like to use it as well :D

**Reply by pmbrull** [reply]
cc @akash-jain-10 let's see if we can work on this starting Jan

**Reply by snayakula** [reply]
Hi, I am looking out for this info. Is the Documentation ready for Deployment on AKS? Could you please share any details about it?

**Reply by degox86** [reply]
Hi,  I'm also interested in Documentation for deploying OpenMetaData on AKS,  any details would help,  thank you
